src/test/java/jenkins/plugins/logstash/LogstashConfigurationMigrationTest.java

```diff
@@ -52,7 +52,7 @@ public void setup()
   {
     mockStatic(LogstashInstallation.class);
     configFile = new File("notExisting.xml");
-    when(LogstashInstallation.getLogstashDescriptor()).thenReturn(descriptor);
+    when(LogstashInstallation.getLogstashDescriptor()).thenAnswer(invocationOnMock -> descriptor);
```

Comment:
some weird powermock issue (https://github.com/powermock/powermock/issues/992), this fix worked